app/models/invoice_item.rb

```diff
@@ -5,7 +5,8 @@ class InvoiceItem < ApplicationRecord
   belongs_to :item
 
   def invoice_creation_date
-    date = invoice.created_at
-    date.strftime("%A, %B %d,%Y")
+    invoice.creation_date_formatted
+    # date = invoice.created_at
```

Comment:
This can be removed since you created the `creation_date_formatted ` method.

Reply:
ah, yes. I had left that commented out in case we had thought that wasn't a good place to abstract the logic. I'll remove that now.

Reply:
Changes have been appended to the PR.